src/coreComponents/physicsSolvers/multiphysics/HydrofractureSolver.cpp

```diff
@@ -920,8 +924,8 @@ real64 HydrofractureSolver< POROMECHANICS_SOLVER >::setNextDt( real64 const & cu
   {
     nextDt = m_surfaceGenerator->getTimestepRequest() < 1e99 ? m_surfaceGenerator->getTimestepRequest() : currentDt;
   }
+  GEOS_LOG_LEVEL_INFO_RANK_0( logInfo::SolverTimeStep, this->getName() << ": nextDt request is "  << nextDt );
```

Comment:
I think you might create a `logInfo::NextDT` (lvl=3, desc about the changing DT) that target only this kind of call, and the other one would fallback to `logInfo::TimeStep` (lvl=1)